core/print_api.php

```diff
@@ -405,10 +405,7 @@ function print_tag_option_list( $p_bug_id = 0 ) {
 	echo '<option value="0">', string_html_specialchars( lang_get( 'tag_existing' ) ), '</option>';
 	foreach ( $t_rows as $t_row ) {
 		$t_string = $t_row['name'];
-		if( !empty( $t_row['description'] ) ) {
-			$t_string .= ' - ' . utf8_substr( $t_row['description'], 0, 20 );
-		}
-		echo '<option value="', $t_row['id'], '" title="', string_attribute( $t_row['name'] ), '">', string_attribute( $t_string ), '</option>';
+		echo '<option value="', $t_row['id'], '" title="', string_attribute( $t_row['description'] ), '">', string_attribute( $t_string ), '</option>';
```

Comment:
Assignment of $t_string in line 407 is no longer needed.
You could use now $t_row['name'] in line 408.